frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -74,9 +74,10 @@ const HeaderActions: React.FC<IHeaderActions> = ({
         <>
           <Link
             style={{ width: 95 }}
-            onClick={() => {
+            onClick={e => {
               jumpToView();
               if (onLinkClick) onLinkClick();
+              e.stopPropagation();
             }}
```

Comment:
added this to prevent clicking on the "View Details" link in the property info slideout from propagating to the map and highlighting the parcel directly behind the click.